docs/proposal/203-duffle-build.md

```diff
@@ -1,5 +1,5 @@
 # Duffle Build
 
-This document describes how `duffle build` works, and how it uses the duffle build file: `duffle.json, duffle.yaml, duffle.toml`.
+This document describes how `duffle build` works, and how it uses the duffle build configuration file: `duffle.json, duffle.yaml, duffle.toml`.
```

Comment:
I think it would be better to use the same term everywhere. Also, the "build" part is implied by the context.
```suggestion
This document describes how `duffle build` works, and how it uses the duffle configuration file: `duffle.json, duffle.yaml, duffle.toml`.
```